flink-end-to-end-tests/test-scripts/test_ha_datastream.sh

```diff
@@ -98,6 +98,6 @@ function run_ha_test() {
 STATE_BACKEND_TYPE=${1:-file}
 STATE_BACKEND_FILE_ASYNC=${2:-true}
 STATE_BACKEND_ROCKS_INCREMENTAL=${3:-false}
-ZOOKEEPER_VERSION=${4:-3.4}
+ZOOKEEPER_VERSION=${4:-3.5}
```

Comment:
Shall we move that into a variable `DEFAULT_ZOOKEEPER_VERSION` in `common_ha.sh`?

Reply:
as far as I'm concerned these shouldn't have defaults in the first place; but changing these here is less work than doing the error handling.